Lib/test/test_wave.py

```diff
@@ -108,6 +110,26 @@ def test__all__(self):
         blacklist = {'WAVE_FORMAT_PCM'}
         support.check__all__(self, wave, blacklist=blacklist)
 
+    def test_open(self):
```

Comment:
I'd rename this to something like `test_open_pathlike`.

Reply:
This test checks only failed cases. It would be better to test successful opens.

Test not only reading, but writing too.

Reply:
Is it acceptable to add a wave file to the test directory?  I didn't test on a .wav file since even a small one can be 20-30k -- if that's okay then I will add one and test that.

Reply:
There are sample wave files in the audiodata directory. See how other tests use them.